scripts/lib/CIME/case_run.py

```diff
@@ -231,7 +231,7 @@ def resubmit_check(case):
         run_cmd(cmd, verbose=True)
 
     if resubmit:
-        if testcase is not None and testcase in ['ERR']:
+        if testcase is not None and testcase in ['ERR', 'PET']:
```

Comment:
@jedwards4b , this seems pretty hack to me. I had to use pdb to step through the entire submit/run phase of PET to figure out why resubmit wasn't running case.test. I'm wondering why we do this in the first place?

Reply:
It's probably enough to have if testcase is not None:

Reply:
OK, let me do that.